src/routes/prescriptions.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const router = express.Router()
+const {Prescription} = require('../domain/prescription')
+const {logger} = require('../utils/utils')
+const {StateMachine} = require('../state-machine/state-machine')
+const {newBadRequestError} = require('../utils/errors')
+
+router.post('/prescriptions', bodyParser.json(), (req, res, next) => {
```

Comment:
Deberia ser singular `prescription`

Reply:
pocua? no es en plural? 
https://stackoverflow.com/questions/6845772/rest-uri-convention-singular-or-plural-name-of-resource-while-creating-it encontré esto

Reply:
oka, compro